charts/stable/wmbusmeters/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+appVersion: 1.4.1
+description: wmbusmeters helm package
```

Comment:
Please describe what wmbusmeters does here. Often we put a blurb from the project's source repo here.